### PR TITLE
add keydown listener for ProfileModal and AnswerModal

### PR DESCRIPTION
**Pull request by quiz1991**
@anthonyseu @jacgrady1

### CODE REVIEW

File: components/Nav.js [hunk 1]
```diff
@@ -277,6 +280,14 @@ class Nav extends React.Component {
             window.scrollTo(0, 0);
         }
     }
+    handleKeyDown(e) {
+        // ESC key
+        if (e.keyCode === 27) {
```
Comment (jacgrady1) [inline components/Nav.js]: http://www.material-ui.com/#/components/dialog
there is a modal field called modal to handle click outside to close

Comment (quiz1991) [inline components/Nav.js]: it also take care of esc?

Comment (quiz1991) [inline components/Nav.js]: "You can also close this dialog by clicking outside the dialog, or with the 'Esc' key." ok i'll try. 

But this listener is required for Popover rt?

Comment (jacgrady1) [inline components/Nav.js]: what do you mean pop over?

Comment (quiz1991) [inline components/Nav.js]: the dropdown menu is a Popover

**Comment by quiz1991** [comment]
@jacgrady1 remove esc key handler for profilemodal and answermodal

**Pull request merged**